working/augmentation-libraries/feature-specification.md

```diff
@@ -1040,136 +765,28 @@ extension type A(int b) {
 other types in which case the extension type syntax will then be understood by
 users to be a primary constructor for the extension type.*
 
-The extension type's representation object is _not_ a variable, even though it
-looks and behaves much like one, and it cannot be augmented as such. It is a
-compile time error to have any augmenting declaration with the same name as the
-representation object.
+[primary constructors]:
+https://github.com/dart-lang/language/blob/main/working/2364%20-%20primary%20constructors/feature-specification.md
+
+The extension type's representation object is *not* a variable, even though it
+looks and behaves much like one, and it cannot be augmented as such.
+
+It's a **compile-time error** if:
 
-It is a compile time error if:
+*   An augmenting declaration has the same name as the representation object.
```

Comment:
representation object -> representation variable

(The representation object is the value of the representation variable, only the latter has a *name*.)